### ISSUE TITLE
import PySide.QtGui + ImageQt -> crash on end of program

### ISSUE THREAD

**Issue by moeb** [python-pillow/Pillow#663]
I get a Backtrace with alternating errors when importing PySide.QtGui and ImageQt.
Some errors to get an impression:
```
Error in `python': double free or corruption (out)
Error in `python': free(): invalid pointer
Error in `python': munmap_chunk(): invalid pointer
```

system information:
running in a virtual env on linux 64bit
Python 3.4.0
Pillow 2.4.0
PySide 1.2.2

run this to repeat
```
#!/usr/bin/env python
import sys
import PySide
sys.modules['PyQt4'] = PySide

import PySide.QtGui
import PIL.ImageQt
```

**Comment by radarhere** [comment]
I'm able to replicate this bug on OS X with Python 3.4. I took the line that imports ImageQt and removed parts of the code to get down to the minimum case to trigger the problem.

    import sys
    import PySide
    sys.modules['PyQt4'] = PySide
    import PySide.QtGui
    from PyQt4.QtGui import qRgba

While my knowledge of memory and pointer management is not great, on a surface level, I look at this code and think that the problem probably exists because the same module is being imported under different names.

However, this was presumably a workaround for lack of support for PySide in ImageQt. But since this issue was opened, PySide is now supported in ImageQt. So this is no longer necessary.

So in conclusion, it's not a problem with Pillow, and it's not a use case that Pillow users need to enter into.
